config/crd/bases/consul.hashicorp.com_servicedefaults.yaml

```diff
@@ -95,7 +95,6 @@ spec:
               type: string
           type: object
         status:
-          description: ServiceDefaultsStatus defines the observed state of ServiceDefaults
```

Comment:
I could get this back by adding a doc block to the Status struct but it seemed a bit unnecessary?